phpstan.neon.dist

```diff
@@ -0,0 +1,9 @@
+includes:
+    - phpstan-baseline.neon
+
+parameters:
+    reportUnmatchedIgnoredErrors: false
+
+    level: max
```

Comment:
i'd prefer to start with level 1 to see the important things. if we start with max right away, the baseline is huge and priorities are not clear.